### ISSUE TITLE
 The preview window doesn't seem to inherit the option

### ISSUE THREAD

**Issue by emxxjnm** [nvim-neo-tree/neo-tree.nvim#562]
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/e968cda658089b56ee1eaa1772a2a0e50113b902/lua/neo-tree/ui/popups.lua#L39

**Comment by cseickel** [comment]
What option did you want it to inherit?

**Comment by emxxjnm** [comment]
Sorry to confuse you, I meant wouldn't it be better to use `vim.tbl_deep_extend` here?

https://github.com/nvim-neo-tree/neo-tree.nvim/blob/v2.x/lua/neo-tree/ui/popups.lua#L62
![image](https://user-images.githubusercontent.com/54089360/195029842-941a089c-cdb9-4e2d-b2e2-217fe4980c1f.png)

**Comment by cseickel** [comment]
> Sorry to confuse you, I meant wouldn't it be better to use `vim.tbl_deep_extend` here?

No, I don't think that would be a good idea at all. Most of the top level options are sets of options that should not be arbitrarily merged.

What option did you want it to inherit that is not being inherited now?
